charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-configmap.yaml

```diff
@@ -482,7 +471,7 @@ data:
       elseif severity == "W" or severity == "WRN" or severity == "WARN" or severity == "WARNING" then
         record["severity"] = "WARN"
         modified = true
-      elseif severity == "E" or severity == "ERR" or severity == "ERROR" then
+      elseif severity == "E" or severity == "ERR" or severity == "ERROR" or severity == "eror" then
```

Comment:
Why is that needed?

Reply:
grafana

Reply:
t=2020-08-17T09:40:08+0000 lvl=eror msg="..." logger=context error="..."

Reply:
Several lines above there is `      severity = trim(severity):upper()`. I guess comparing to lowercase log level won't catch anything.

Reply:
yup, i will change it to `EROR`